portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/handlers/SiteHandler.java

```diff
@@ -556,6 +556,10 @@ public void doSite(HttpServletRequest req, HttpServletResponse res, Session sess
 		rcontext.put("showFavStarsOnAllFavSites",ServerConfigurationService.getBoolean(SAK_PROP_SHOW_FAV_STARS_ON_ALL, SAK_PROP_SHOW_FAV_STARS_ON_ALL_DFLT));
 		
 		addLocale(rcontext, site, session.getUserId());
+
+		addServerTime(rcontext);
+
+		addUserTimezoneOffset(rcontext);
```

Comment:
Just one method called addUserTimezoneInfo ? And then we add 3-4 new vars around time?